api/ruby/2fa_checker.rb

```diff
@@ -1,16 +1,33 @@
 require 'octokit.rb'
 
-if ARGV.length != 1
-  $stderr.puts "Pass in the name of the organization you're interested in checking."
+begin
+  ACCESS_TOKEN = ENV.fetch("GITHUB_TOKEN")
+  HOSTNAME = ENV.fetch("GITHUB_HOSTNAME")
+rescue KeyError
+  $stderr.puts "To run this script, please set the following environment variables:"
+  $stderr.puts "- GITHUB_TOKEN: A valid access token with Organzation admin priviliges"
+  $stderr.puts "- GITHUB_HOSTNAME: A valid GitHub Enterprise hostname"
   exit 1
 end
 
-# !!! DO NOT EVER USE HARD-CODED VALUES IN A REAL APP !!!
-# Instead, set and test environment variables, like below
-client = Octokit::Client.new(:access_token => ENV['MY_PERSONAL_TOKEN'])
+Octokit.configure do |kit|
+  kit.api_endpoint = "#{HOSTNAME}/api/v3"
+  kit.access_token = ACCESS_TOKEN
+  kit.auto_paginate = true
+end
+
+if ARGV.length != 1
```

Comment:
Should this check for length > 0? What happens if i don't pass in a value at all?